packages/yoastseo/src/scoring/assessments/seo/ProductSKUAssessment.js

```diff
@@ -74,15 +78,23 @@ export default class ProductSKUAssessment extends Assessment {
 			( [ "simple", "external" ].includes( customData.productType ) || customData.hasVariants === false ) ) {
 			return false;
 		}
-
+		// Assessment is not applicable if we don't want to assess variants and the product has variants.
+		if ( this._config.assessVariants === false ) {
```

Comment:
We need to look again whether the `assessVariants` has become obsolete now. If I understand correctly, this was initially used to distinguish between shopify and woo.

Reply:
Or at least, the purpose is not really clear to me.